packages/cli/core/src/commands/org/resources/__snapshots__/list.spec.ts.snap

```diff
@@ -0,0 +1,26 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
I moved everything into the snap file to make the spec file lighter.

Reply:
Nice